legion/legion/external/edi.py

```diff
@@ -169,7 +171,8 @@ def deploy(self, image, count=1, livenesstimeout=2, readinesstimeout=2):
         :return: list[:py:class:`legion.containers.k8s.ModelDeploymentDescription`] -- affected model deployments
         """
         payload = {
-            'image': image
+            'image': image,
+            'model_iam_role': model_iam_role
```

Comment:
actually the role will be set in most cases, but even if by some reason someone will try to deploy with None role I guess it will be better to see in annotation that the IAM role is set as None. This might be useful for debug and in order to eliminate errors.